includes/Integrations/Elementor/Subscription_Plans_Widget.php

```
@@ -0,0 +1,1506 @@
+	 * Render the subscribe button
+	 *
+	 * @since WPUF_SINCE
+	 *
+	 * @param \WP_Post $pack              The subscription plan post object.
+	 * @param string   $current_pack_id   Current active pack ID.
+	 * @param string   $current_pack_status Current pack status.
+	 * @param float    $billing_amount    The billing amount.
+	 * @return void
+	 */
+	protected function render_button( $pack, $current_pack_id, $current_pack_status, $billing_amount ) {
+		$user_id = get_current_user_id();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$user_id'. (undefined)

(UnusedLocalVariable)